build.gradle.kts

```diff
@@ -37,6 +42,10 @@ version = properties("pluginVersion")
 // Configure project's dependencies
 repositories {
     mavenCentral()
+    // this part is mandatory for all modules for platform version 2
+    intellijPlatform {
```

Comment:
**nitpick**: a link to the appropriate documentation here would be nice to have.